cmd/draft/create.go

```diff
@@ -26,6 +28,8 @@ const (
 `
 )
 
+var PackNotFound = errors.New("Could not find a pack Q_Q")
+
```

Comment:
Any reason this is exported?

Reply:
I actually wanted to move this over to pkg/draft/pack. Good catch!